spec/models/post_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'is valid with a title, comments_counter, likes_counter, author_id and author' do
+    user = User.create(name: 'John Doe')
+    post = Post.new(title: 'Lorem ipsum', comments_counter: 0, likes_counter: 0, author_id: user.id, author: user)
+    expect(post).to be_valid
+  end
+
+  it 'is invalid without a title' do
+    user = User.create(name: 'John Doe')
+    post = Post.new(comments_counter: 0, likes_counter: 0, author_id: user.id, author: user)
+    expect(post).to_not be_valid
+  end
+
+  it 'is invalid with a negative comments_counter' do
+    user = User.create(name: 'John Doe')
+    post = Post.new(title: 'Lorem ipsum', comments_counter: -1, likes_counter: 0, author_id: user.id, author: user)
+    expect(post).to_not be_valid
+  end
+end
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the requirements:

![image](https://github.com/sergio-usma/rails_blog_pj/assets/77069011/f39b7d38-3f08-4cf1-9be9-365c7945edd6)

Please make sure to include tests for ALL your model's methods (`recent_comments`, `update_post_counter`) and all the validations: 

![image](https://github.com/sergio-usma/rails_blog_pj/assets/77069011/39579a7b-a64f-4521-8703-79aabe6b7242)

I can't see a test for the title's length and likes counter.

- [ ] **OPTIONAL** When you're testing validations, is often a good practice to always test for both negative and positive cases, for example, if you're testing the `title's presence`, write a test to describe what happens if the title is missing and what happens if not. 🤓 🔧

Reply:
This change has been done well